backup/crowbar-backup

```diff
@@ -88,12 +88,8 @@ function check_resolv_conf
 {
   # check if DNS servers are available and can resolve domain names
   for ns in `grep "^\s*nameserver" /etc/resolv.conf | sed -e "s/\s*nameserver \([[:digit:]\.]\+\)[^[:digit:]\.]*/\1/" ` ; do
-    if ! netcat -zu $ns 53 ; then
-      echo "Error: Nameserver '$ns' is not available. Please reconfigure /etc/resolv.conf manually."
-      exit 1
-    fi
-    if ! getent hosts "$HOSTTEST" >/dev/null; then
-      echo "Error: Nameserver '$ns' can not resolve or ping $HOSTTEST. Please reconfigure /etc/resolv.conf manually"
+    if ! dig @$ns +short "$HOSTTEST" >/dev/null; then
```

Comment:
SLE12
doener3:~ # dig @1.1.1.1 +short www.suse.com
;; connection timed out; no servers could be reached
doener3:~ # echo $?
9
doener3:~ # dig @8.8.8.8 +short www.suse.com                                                                                                                                                                                                                                     
130.57.66.10
doener3:~ # echo $?
0
doener3:~ #